css/lists.css

```diff
@@ -24,3 +24,35 @@
 .spaced-list li {
 	margin: 1rem;
 }
+
+.dropdown li {
+	width: 1rem !important;
+	position: absolute;
+}
+
+.dropdown-list {
+	display: flex;
+	flex-direction: column;
+	gap: 2rem;
+	align-items: center;
+	width: 100%;
+	height: auto;
+	position: relative;
+	bottom: -5rem;
+	left: -6rem;
+	z-index: var(--max-z-index);
+	display: none;
+}
+
+.dropdown-list > li {
+	width: 100%;
+	padding: 0;
+	margin: 0;
+}
+.dropdown:hover .dropdown-list {
+	display: flex;
+}
+
+.dropdown:hover {
+	display: flex !important;
+}
```

Comment:
here as well. I guess increasing the specificity can do the job

Reply:
ok will change that